openqa-label-known-issues

```diff
@@ -104,7 +110,16 @@ label_on_issues_from_issue_tracker() {
     # splice each line with a call to label_on_issue
     echo "$issues" | (while read -r issue; do
         read -r subject
-        after=${subject#*\"} && search=${after%\"*} && [[ ${#search} -ge $min_search_term ]] && label_on_issue "$id" "$search" "poo#$issue $subject" "${after//*\":retry*/1}" && break; done)
+        after=${subject#*\"}
+        search=${after%\"*}
+        if [[ ${#search} -ge $min_search_term ]]; then
+            if [[ $after =~ :force_result:([a-z_]+) ]]; then
+                force_result_on_issue "$id" "poo#$issue" "${BASH_REMATCH[1]}"
+            fi
+            label_on_issue "$id" "$search" "poo#$issue $subject" "${after//*\":retry*/1}" "$force_result" && break
```

Comment:
Should we turn this around so that in case posting this fails at least is tried first?

Reply:
> Should we turn this around so that in case posting this fails at least is tried first?

I fail to parse this sentence...

Reply:
Nevermind. Just leave it like it is for now. We will see how it goes :)